BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Views/SkontoAmountToPayView.swift

```diff
@@ -132,6 +171,19 @@ class SkontoAmountToPayView: UIView {
         currencyLabel.isHidden = !isEditable
     }
 
+    func updateValidationMessage(_ message: String) {
+        guard validationLabel.isHidden else {
+            return
+        }
+        validationLabel.text = message
+        validationLabel.isHidden = false
+    }
+
+    func hideValidationMessage() {
+        validationLabel.text = ""
```

Comment:
[clarification] Why setting to nil is not good here instead of empty string?

Reply:
here's no matter `nil` or "", because we are not using anywhere check for nil or empty string. Its just to be sure that previous value is not showed to a user